java-client/src/main/java/org/opensearch/client/transport/httpclient5/ResponseException.java

```diff
@@ -92,4 +92,12 @@ static String buildMessage(Response response) throws IOException {
     public Response getResponse() {
         return response;
     }
+
+    /**
+     * Status code returned by Elasticsearch. Shortcut for
+     * {@code response().status()}.
+     */
+    public int status() {
```

Comment:
Q: Should it be `status` or `getStatus`? What's the convention?

Reply:
i took this 1:1 from `OpenSearchException`, presuming that this was the accepted convention. happy to change it to `getStatus`

Reply:
i've left it at `status` for now pending your feedback